app/views/accounts/show.html.erb

```diff
@@ -9,8 +9,20 @@
 <%= render @view_model.pending_profile_partial, view_model: @view_model %>
 <%= render 'accounts/header', view_model: @view_model %>
 
+<h2><%= t('account.index.email_preferences') %></h2>
+
 <div class="mb3 card profile-info-box">
   <%= render 'emails' %>
+
+  <h2><%= t('i18n.language') %></h2>
```

Comment:
Based on the content hierarchy, I'd expect this to be an `h3`, as a subheading of the "Email Preferences" section heading. 

This can be problematic for people relying on content hierarchy through something like VoiceOver web rotor:

![image](https://user-images.githubusercontent.com/1779930/97889520-74af9d80-1cfa-11eb-849b-48bfa45e4057.png)

While it moves us further away from migrating off BassCSS, a short-term resolution could be to use the `h2` class to style the `h3` heading as it is today:

```suggestion
  <h3 class="h2"><%= t('i18n.language') %></h3>
```

Ideally we could just use `h3`, but the default styling for `h3` has it appear as slightly larger than the `h2` 🤷 

The same goes for the previous heading "Email addresses".

Reply:
Good point, I'll scope the headings better